.github/workflows/performance_harness_run.yaml

```diff
@@ -12,10 +22,102 @@ defaults:
     shell: bash
 
 jobs:
-  tmp:
-    name: Stub
+
+  v:
+    name: Discover Inputs
+    runs-on: ubuntu-latest
+    outputs:
+      test-params: ${{steps.overrides.outputs.test-params}}
+    steps:
+      - name: Setup Input Params
+        id: overrides
+        env:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
this used any where?

Reply:
Addressed: [8c47cce](https://github.com/AntelopeIO/leap/pull/1378/commits/8c47cce233698dd4801896b7727eeb09188057a9)